management/web_update.py

```diff
@@ -23,7 +23,10 @@ def get_web_domains(env, include_www_redirects=True, exclude_dns_elsewhere=True)
 		# to the main domain for. We'll add 'www.' to any DNS zones, i.e.
 		# the topmost of each domain we serve.
 		domains |= set('www.' + zone for zone, zonefile in get_dns_zones(env))
-	 
+
+	# add the required subdomains for MTA-STS
+	domains |= set('mta-sts.' + zone for zone, zonefile in get_dns_zones(env))
```

Comment:
get_dns_zones should probably be get_mail_domains. I haven't read the standard yet but I'm guessing it's based on the domain in the email address. get_dns_zones returns a slightly different list of domains the box serves.

Reply:
Fixed in https://github.com/mail-in-a-box/mailinabox/pull/1556/commits/4968f3dc1086e70ce11462093f0bc578a5d3d81b